common/build.gradle

```diff
@@ -147,10 +147,9 @@ dependencies {
     testImplementation 'org.json:json:20160810'
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
-    testImplementation "com.google.dexmaker:dexmaker-mockito:1.2"
     testImplementation "org.powermock:powermock-module-junit4:$rootProject.ext.powerMockVersion"
     testImplementation "org.powermock:powermock-module-junit4-rule:$rootProject.ext.powerMockVersion"
-    testImplementation "org.powermock:powermock-api-mockito:$rootProject.ext.powerMockVersion"
+    testImplementation "org.powermock:powermock-api-mockito2:$rootProject.ext.powerMockVersion"
```

Comment:
What is mockito2? How is it different from mockito?

Reply:
Mockito 2.0+ is a breaking change from Mockito 1.0 series.  Powermock needs a different version for this, AFAIK.

Reply:
https://mvnrepository.com/artifact/org.powermock/powermock-api-mockito2